db/schemas/manifestschema.json

```diff
@@ -0,0 +1,356 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id":"https://e-manifest.18f.gov/schemas/manifestschema.json",
+    "title": "e-Manifest JSON Schema",
+    "description": "Second draft of JSON Schema.  This version, changed file name, enforced snake_case, and removed minlengths. I fully expect this to be edited and revised.  Resource used to make this was http://spacetelescope.github.io/understanding-json-schema/index.html",
+
+    "definitions":{
+        "address": {
+             "type": "object",
+            "properties":{
+                "address_1":   {
+                    "description": "Address line 1",
+                    "type": "string",
+                    "maxLength":30
+                },
+                "address_2":   {
+                    "description": "Address line 2",
+                    "type": "string",
+                    "maxLength":30
+                },
+                "city":   {
+                    "description": "City name",
+                    "type": "string",
+                    "maxLength":25
+                },
+                "state":   {
+                    "description": "Two character abbreviation for State or Province",
+                    "type": "string",
+                    "maxLength":2,
+                    "minLength":2, 
+                    "COMMENT":" add Enummerated value"
+                },
+                "zip_code":   {
+                    "description": "Postal Code",
+                    "type": "string",
+                    "maxLength":14, 
+                    "COMMENT":" add Regular Expression"
+                }
+            }
+        },
+
+        "telephonenumberdatatype":  {
```

Comment:
OK, but we need a plan to make this change.  The name included is the name of the field in industry.html and the one posted in our sample.  Let's discuss at story time.  FWIW, I am not opposed to changing the names much.  We'll be adding new fields anyway.